src/applications/disability-benefits/new-686/config/constants.js

```diff
@@ -1,5 +1,5 @@
 export const textOnlyPattern = '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*0-9]+$';
-export const numberOnlyPattern = '^[0-9]*$';
+export const numberAndDashPattern = '^[0-9]*[-]*[0-9]*[-]*[0-9]*$';
```

Comment:
I see what the intention is. This allows for invalid phone number entry in the veteran address. Something like `-123-4567` or `---345--32` ends up being 'valid.' I took a look at what's in `vets-json-schema` for a number of other forms, and it seems many of them follow this pattern: `{type: 'string', pattern: "^\\d{10}$"}`. We should probably stick to common convention here and create a new `phonePattern` const with the above pattern as it's value.

Reply:
Yep, totally agree. See comment above for more info